.devcontainer/README.md

```diff
@@ -12,37 +12,47 @@ All configuration related to the development container is in the `.devcontainer`
 ### Setup
 
 - Install Docker Desktop or Docker CE on a host machine. The host machine can be Windows, Linux, or Mac, and can run on-premises or in the cloud.
-- Recommended: If you are using Windows for the host machine, clone and open this repository on Windows Subsystem for Linux (WSL). WSL is not required, but disk IO performance in the container is better and the overall experience is more consistent when running a development container from WSL. 
-
-    > NOTE: the Windows Git Credential Manager can be configured to work from WSL. See the [documentation here](https://docs.microsoft.com/en-us/windows/wsl/tutorials/wsl-git#git-credential-manager-setup). Below is the command to run for setting the Windows credential manager in WSL:
+- Clone the Mission LZ from GitHub to a local workspace on the host machine. 
+  - **Recommended:** If you are using Windows for the host machine, clone and open this repository on Windows Subsystem for Linux (WSL). WSL is not required, but disk IO performance in the container is better and the overall experience is more consistent when running a development container from WSL.
+    > **NOTE:** When using Windows with WSL as the host machine, we recommend the following additional steps:
+    > - After [installing WSL for Windows](https://docs.microsoft.com/en-us/windows/wsl/install-win10), you can run your Linux distribution path right inside the Windows command prompt (or in PowerShell), by entering `wsl.exe` or `bash.exe`. These commands will switch to a display of the Linux command line, using the path for your current directory. This path will appear to be in a mounted folder, `/mnt/c`, because we're now viewing your Windows C:\ drive folder from the Linux subsystem. You can access all of your local computer's file system from within the Linux shell by using this `/mnt/c` mounted file path.
```

Comment:
In addition to running wsl.exe from a command or Powershell window, Windows Terminal is a good option for running WSL. It detects all WSL instances and gives you a separate shell for each one.

Reply:
Good call out. Should this be added?

Reply:
I think we can either give them the advice above plus Windows Terminal, or leave out all advice related to how to configure and launch WSL. Either is good I think.